src/openlcb/DatagramDefs.hxx

```diff
@@ -128,7 +132,15 @@ struct DatagramDefs
     {
         return error & BUFFER_UNAVAILABLE;
     }
-    
+
+    /** Constants used by the protocol_size function. */
```

Comment:
This change is suspicious.  Is it really required?  Why?

Reply:
this was necessary due to compilation failure with it in private. I'd like to revisit this further.